lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -154,10 +154,7 @@ def convert_zunits(self, z):
 
     def _process_unit_info(self, xdata=None, ydata=None, zdata=None,
                            kwargs=None):
-        """
-        Look for unit *kwargs* and update the axis instances as necessary
-
-        """
+        """Look for unit *kwargs* and update the axis instances as necessary."""
```

Comment:
This exceeds the line limit of 79 chars and thus flake8 complains. One option would be less wordy:
~~~
"""Update the axis instances based on unit *kwargs* if given."""
OTOH it would be even better to document the parameters.
~~~

Reply:
Thank you for pointing that out! I changed the docstring so that it does not exceed the line limit as you advised. In the following days I will make another PR to expand on the doc for `_3d_extend_contour` and for `_process_unit_info` so that the parameters are documented.